libs/core/src/components/segmented-control/segmented-control.style.css

```diff
@@ -5,9 +5,9 @@
   box-sizing: border-box;
   contain: layout;
   display: inline-flex;
-  gap: 0.25rem;
   height: 3rem;
```

Comment:
I can see here we are using rem's, should we try to switch other components to using rem units or this to PX ?

Reply:
Typical best practice is to use rem's for sizing stuff. That way, when a user has changed the base font-size in their browser, other stuff will follow proportionally. This needs more testing though!